doc/src/documentation_style_guide.rst

```diff
@@ -0,0 +1,1019 @@
+path in Windows, and run in your shell to navigate to the folder. (Also note
+that Linux uses ``/`` instead of ``\`` for file paths.)
+
+This method provides better compatibility than Cygwin or MSYS2 and more
+convenience than a virtual machine if you partially need a Linux environment
+for your workflow, however this method is only viable for Windows 10 64-bit
+users.
+
+2. Build the Documentation
+--------------------------
+
+The documentation can be built by running the ``makefile`` in the ``doc``
+subdirectory.
+
+To start, in your preferred web browser, use the drop down menu and select
+“open file” to navigate into the sympy/doc folder saved on your computer. In
+the doc folder, select the _build folder, then the html folder, and in the html
+folder, open the index.html file.
+
+To build the HTML documentation, run::
+
+  cd doc
+
+  make html
+
+This builds a local version of the documentation in ``doc/_build/html`` in your
+web browser.
+
+Open ``_build/html/index.html``.
+
+3. Make a Contribution
+----------------------
+
+For in-depth instructions on how to contribute to SymPy’s code base including
+coding conventions, creating your environment, picking an issue to fix, and
+opening a pull request, please read our full `Development Workflow
+<https://github.com/sympy/sympy/wiki/Development-workflow>`_ guide.
+
+Best Practices for Writing Documentation
+========================================
+
+Please follow these formatting, style, and tone preferences when writing
+documentation.
+
+Formatting Preferences
+----------------------
+
+**Math vs. Code**
+
+**Math:** text that is surrounded by dollar signs $ _ $ will be rendered as
+LaTeX math. Any text that is meant to appear as LaTeX math should be written as
+$math$. In the HTML version of the docs, MathJax will render the math.
+
+**Example**
+
+::
+
+  The Bessel $J$ function of order $\nu$ is defined to be the function
+  satisfying Bessel’s differential equation.
+
+**Code:** text that should be printed verbatim, such as code, should be
+surrounded by a set of double backticks ``like this``. (WORK IN PROGRESS: change
+single backticks to render as code via issue `#13519
+<https://github.com/sympy/sympy/issues/13519>`_.)
+
+**Example**
+
+::
+
+  To use this class, define the ``_rewrite`` and ``_expand`` methods.
+
+Sometimes a variable will be the same in both math and code, and can even
+appear in the same paragraph, making it difficult to know if it should be
+formatted as math or code. If the sentence in question is discussing
+mathematics, then LaTeX should be used, but if the sentence is discussing the
+SymPy implementation specifically, then code should be used.
+
+**Example**
+
+::
+
+  def stirling(n, k, d=None, kind=2, signed=False):
+      """
+      ...
+
+      The first kind of Stirling number counts the number of permutations of
+      ``n`` distinct items that have ``k`` cycles... This counts the ways to
```

Comment:
The purpose of this example was to show how sometimes a variable can be both math and code and appear within the same paragraph, once as code and once as math, so here the first instance of ``n`` and ``k`` are meant to render as code. If anyone has better/additional examples of this I'd love to add some more!

Reply:
@mgeier is referring to a rule that I was not aware of, which is that variables that are also parameters to the function should be rendered in italics instead of code. This was also discussed at https://github.com/sympy/sympy/issues/13519. Let's talk about this on the meeting on Monday.

Reply:
@lglattly I don't think that's a good example for showing this, I would format all "variables" mentioned here as function arguments, i.e. `*n*`, `*k*` and `*d*`.

The "variable" *d* isn't even used in the equations (which were omitted here), see https://docs.sympy.org/latest/modules/functions/combinatorial.html#sympy.functions.combinatorial.numbers.stirling.

Reply:
We can replace this with a better example. Do you have a suggestion?

Reply:
@lglattly No, sorry.